mk/protobuf.mk

```diff
@@ -0,0 +1,83 @@
+# Copyright 2014 The Cockroach Authors.
```

Comment:
Let's place this in the `build` directory where other build related stuff lives.